pom.xml

```diff
@@ -66,19 +66,21 @@
       explicitly specifying the latest version here because one we get from the parent POM
       tends to lag behind a bit
     -->
-    <maven-hpi-plugin.version>1.95</maven-hpi-plugin.version>
+    <maven-hpi-plugin.version>1.106</maven-hpi-plugin.version>
```

Comment:
@KostyaSha just seen your #75 comment. Does that not contradict your previous point about changing the minimal jenkins core version, or is there else something special going on in this case?